profile_bluesky/startup/instrument/devices/lambda_750k.py

```diff
@@ -355,21 +355,21 @@ def getCounts(self):
         # pseudo-counter named "ccdc" (that's what chk_ccdc() does)
         if self.chk_ccdc:
             if dm_pars.compression.get() == 1:
-                return self.immout.num_pixels.get()
+                return self.imm1.num_pixels.get()
```

Comment:
will be part of #274

Reply:
and below, as well

Reply:
Got it. Should I merge now and continue to work on it via #274 later?